tests/add-user.test.js

```diff
@@ -89,4 +89,45 @@ describe("Given Item Store is connected to NATS", () => {
             });
         });
     });
+    describe("When I add a new User with a Validation Error [a field (user_id) is wrong-typed]", () => {
+        let response;
+        let message;
+
+        beforeAll(async () => {
+            response = await natsConnection.request(
+                "item-store.add_user",
+                jsonCodec.encode({
+                    user_id: "azerty",
+                    inventory_address: "KT1_SFFG345FFSFdqsfz"
+                })
+            );
+
+            message = jsonCodec.decode(response.data).error;
+        });
+
+        it("Then returns an error", () => {
+            expect(message).toEqual("user_id must be an integer.");
+        });
+    });
+    describe("When I add a new User with a Validation Error [a field (inventory_address) is missing]", () => {
```

Comment:
same here, this test is invalid, an inventory_address can't be assigned by the user, it's assigned by the tokenization service